src/lib/OpenEXRCore/OpenEXRVersion.h

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
the other include files in OpenEXRCore are snake_case. In my twisted mind, snake_case == C, CamelCase == C++, not sure if we want to keep with that?

Reply:
I needed OpenEXRCore to be able to see the file without reaching laterally into another directory which is why it's here; and then I couldn't parse in my mind, is this file "more like" OpenEXRConfig.h, or is it "more like" something that should be here. My first thought had been to put it in "OpenEXR", but "OpenEXR" is higher level than core, so it seemed like opening the door to an unintended dependency cycle (Core needs EXR needs Core).

I'm happy to make it snakecase to tell a cleaner story.